### PR TITLE
docs: fix grammar in ngmodules vs js modules guide

### PR DESCRIPTION
**Pull request by selangley-wa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/guide/ngmodule-vs-jsmodule.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ though they organize it differently, Angular apps rely on both.
 
 ## JavaScript modules
 
-In JavaScript, modules are individual files with JavaScript code in them. To make what’s in them available, you write an export statement, usually after the relevant code, like this:
+In JavaScript, modules are individual files with JavaScript code in them. To make what’s in them available, you write an export statement, usually before the relevant code, like this:
```
Comment (brandonroberts) [inline aio/content/guide/ngmodule-vs-jsmodule.md]: The current statement is correct because you don't write the export before the code you're exporting.

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Update git commit message to `docs: fix grammar in ngmodules vs js modules guide`

File: aio/content/guide/ngmodule-vs-jsmodule.md (hunk 1 above)
Comment (selangley-wa) [reply aio/content/guide/ngmodule-vs-jsmodule.md]: How so? In the example:

  `export class AppComponent {...`

the word export comes before the word class, which comes before the word AppComponent?

Comment (brandonroberts) [reply aio/content/guide/ngmodule-vs-jsmodule.md]: Agreed. The original sentence is referring to code such as:

```ts
class AppComponent {}

export { AppComponent }
```

But is contradictory to the example, so it is valid in this case

Comment (selangley-wa) [reply aio/content/guide/ngmodule-vs-jsmodule.md]: So, the use of **like this:** means the example needs to be changed, or perhaps a 2nd example is needed.

To make what’s in them available, you write an export statement, usually after the relevant code, **like this:**

Comment (brandonroberts) [reply aio/content/guide/ngmodule-vs-jsmodule.md]: I think just changing it to `after` is sufficient rather than adding a new example

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
@selangley-wa You must [Sign the CLA](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#cla) before this PR can move forward.

**Comment by selangley-wa** [comment]
You can make the change if you want.  Getting the state government agency I work for to sign a contributor license is WAY too much trouble.

**Comment by brandonroberts** [comment]
Ok. We'll get a new PR open then. Thanks for the contribution either way.

**Pull request closed without merging**